XRTK-Core/Packages/com.xrtk.core/Extensions/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using XRTK.Interfaces;
+
+namespace XRTK.Extensions
+{
+    public static class ConfigurationExtensions
+    {
+        /// <summary>
+        /// Attempts to get the name of the first configuration?
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="configurations"></param>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public static bool TryGetConfigurationName<T>(this IMixedRealityServiceConfiguration[] configurations, out string name) where T : IMixedRealityService
```

Comment:
Not used? I'm just going to remove this.

Reply:
I wouldn't remove this as I can see a use for this in projects, especially in the areas of networking and personal services.